src/features/pilot_management/Roster/index.vue

```diff
@@ -48,66 +48,76 @@
-                v-bind="dragOptions"
-                @start="drag = true"
-                @end="drag = false"
-                @change="moved($event, g)"
-              >
-                <component
-                  :is="pilotCardType"
-                  v-for="(p, i) in pilots.filter(x => x.Group === g)"
-                  :key="`${pilotCardType}_${i}`"
-                  :pilot="p"
-                  :small="profile.GetView('roster') === 'small-cards'"
-                />
-              </draggable>
-            </v-expand-transition>
+            <v-row no-gutters class="pl-10 ml-n12 heading h3 white--text primary sliced">
+              <v-col cols="auto">
+                <v-btn small dark icon class="mt-n1" @click="toggleHidden(g)">
+                  <v-icon v-html="!g.hidden ? 'mdi-chevron-down' : 'mdi-chevron-up'" />
+                </v-btn>
+                {{ g.name ? g.name : 'Ungrouped' }}
+                <span class="overline">({{ g.pilotIDs.length }})</span>
+              </v-col>
+              <v-col v-if="g.name" cols="auto" class="ml-auto mr-8">
+                <v-menu offset-x left :close-on-content-click="false">
+                  <template v-slot:activator="{ on }">
+                    <v-btn dark small icon class="fadeSelect" v-on="on">
+                      <v-icon>mdi-circle-edit-outline</v-icon>
+                    </v-btn>
+                  </template>
+                  <v-card>
+                    <v-card-text>
+                      <v-text-field
+                        :value="g.name"
+                        autofocus
+                        outlined
+                        hide-details
+                        label="Group Name"
+                        @change="setGroupName(g, $event)"
+                      />
+                    </v-card-text>
+                  </v-card>
+                </v-menu>
+                <v-btn dark small icon class="fadeSelect" @click="deleteGroup(g)">
+                  <v-icon>mdi-delete</v-icon>
+                </v-btn>
+              </v-col>
+            </v-row>
+            <div
+              v-if="!g.hidden"
+              :style="profile.GetView('roster') !== 'list' ? 'margin-left: -8px; width: 100vw;' : ''"
+            >
+              <v-expand-transition>
+                <draggable
+                  :list="g.pilotIDs"
+                  :group="{ name: 'group' }"
+                  :disabled="preventDnd"
+                  v-bind="dragOptions"
+                  @start="drag = true"
+                  @end="dragOff()"
+                  @change="movePilot(g.name, $event)"
+                >
+                  <component
+                    :is="pilotCardType"
+                    v-for="(id, j) in g.pilotIDs"
+                    :key="`${pilotCardType}_${j}`"
+                    :pilot="getPilotFromId(id)"
+                    :small="profile.GetView('roster') === 'small-cards'"
+                    :dragging="drag"
```

Comment:
When `forceFallback` was set to `true` in the dragOptions to fix Autoscrolling and a weird offset Ghost icon in Chrome, a new error emerged in Firefox: Whenever the user releases the dragged object, CompCon would treat it like a released click and navigate to the Pilot Sheet.  Had to finagle a short timeout function `dragOff` to keep it from propagating.  If you have recommendations on how to better solve this, please let me know.  I tried `click.stop` and `click.prevent` with no luck, but maybe I was using them the wrong way. :shrug: